benchmarks/BoltzmannWealth/boltzmann_wealth.py

```diff
@@ -46,13 +46,15 @@ def __init__(self, seed=None, n=100, width=10, height=10):
         self.datacollector = mesa.DataCollector(
             model_reporters={"Gini": compute_gini}, agent_reporters={"Wealth": "wealth"}
         )
-        # Create agents
-        for _ in range(self.num_agents):
-            a = MoneyAgent(self)
-            # Add the agent to a random grid cell
-            x = self.random.randrange(self.grid.width)
```

Comment:
I thought `CellCollection` already has `select_random_cell`, and so we have `a.move_to(self.grid.select_random_cell())`?

Reply:
yes, and I used that in the wolf sheep model. But boltzmann benchmark still uses the old grid so I cannot use that here.